### PR TITLE
MA - Added groq for challenge overview

### PR DESCRIPTION
**Pull request by MeenuDhanoa**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ import {
   fetchChildren,
   fetchParentByRailContentId,
   fetchMethodPreviousNextLesson,
-  fetchLiveEvent,
+  fetchChallengeOverview,
```
Comment (adrian-musora) [inline src/index.js:32]: We need this.
```suggestion
  fetchLiveEvent,
  fetchChallengeOverview,
```

File: src/index.js [hunk 2]
```diff
@@ -72,5 +72,5 @@ export {
     fetchChildren,
     fetchParentByRailContentId,
     fetchMethodPreviousNextLesson,
-    fetchLiveEvent,
+    fetchChallengeOverview,
```
Comment (adrian-musora) [inline src/index.js:75]:
```suggestion
    fetchLiveEvent,
    fetchChallengeOverview,
```

File: src/services/sanity.js [hunk 3]
```diff
@@ -886,6 +886,39 @@ export async function fetchPackChildren(railcontentId) {
   return fetchChildren(railcontentId, 'pack');
 }
 
+export async function fetchChallengeOverview(id) {
```
Comment (adrian-musora) [inline src/services/sanity.js:889]: This is an ignorant question that i couldn't find a quick answer to. Is this method intended to only get `Challenge` type content? If so, I'd prefer if we moved the fields to the `contentTypeConfig.js` to maintain the design.

**Review by adrian-musora — CHANGES_REQUESTED** [review]

**Review by csillaj — APPROVED** [review]

**Comment by Joels-code** [comment]
I agree with Adrians points. I will re-review once the changes are made

File: src/contentTypeConfig.js [hunk 4]
```diff
@@ -52,6 +52,12 @@ let contentTypeConfig = {
             '"primary_cta_text": select(dateTime(published_on) > dateTime(now()) && dateTime(enrollment_start_time) > dateTime(now()) => "Notify Me", "Start Challenge")',
             'challenge_state',
             'challenge_state_text',
+            'description',
```
Comment (adrian-musora) [inline src/contentTypeConfig.js:55]: Description is an object field, so has a complex structure. We may update the format later, but for the moment are using the the raw html. 
```suggestion
            `"description": ${descriptionField}`,
```

**Review by adrian-musora — APPROVED** [review]

**Pull request merged**